### DISCUSSION TITLE
Ton Blockchain

### DISCUSSION THREAD

**Discussion by xxargonaut** [community/community#136018] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Please share a tutorial where I can learn how to deploy a simple Ton smart contract to the testnet and integrate it with the front end.

**Comment by 0xpuck** [comment]
hi
I warmly congratulate you on learning Ton

**Reply by xxargonaut** [reply]
long time no see

yeah I wanna study Ton

please help me

**Reply by 0xpuck** [reply]
Study the following in order

https://stepik.org/course
https://stepik.org/lesson

**Comment by 0xpuck** [comment, marked as answer]
> ### Select Topic Area
> Question
> 
> ### Body
> Please share a tutorial where I can learn how to deploy a simple Ton smart contract to the testnet and integrate it with the front end.

For this, please refer to:
https://github.com/OpenProduct
https://github.com/ton-community/tutorials

**Reply by xxargonaut** [reply]
thank you a lot
